Bottom: a180c1def772b86aa11aa8dfc1d0e761311f0caa
Top:    a2ae0663248d13e13725e84b4064e7a28e63fd38
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-03 23:35:57 +0100

Batch size for accumulating lines down to 1

This works around a bug seen on one of my boards, and is similar
to what I had in the non-pours variant of the code.


---

diff --git a/src/pour.c b/src/pour.c
index 7967c68..24846d4 100644
--- a/src/pour.c
+++ b/src/pour.c
@@ -68,7 +68,7 @@ RCSID ("$Id$");
 
 #define UNSUBTRACT_BLOAT 100
 #define SUBTRACT_PIN_VIA_BATCH_SIZE 100
-#define SUBTRACT_LINE_BATCH_SIZE 20
+#define SUBTRACT_LINE_BATCH_SIZE 1
 
 /* ---------------------------------------------------------------------------
  * local prototypes
